src/middlewares/snapshot-middleware.ts

```diff
@@ -0,0 +1,34 @@
+import { PolarisGraphQLContext } from '@enigmatis/polaris-common';
```

Comment:
move to polaris-middlewares and make it a plugin instead of middleware

Reply:
Why plugin

Reply:
We need both the plugin and middleware

Reply:
resolved in teams call

Reply:
resolved in teams call